connector/mqtt/vernemq/broker/README.md

```diff
@@ -1,55 +1,58 @@
 # **Dojot VerneMQ**
 
-The Dojot VerneMQ service is a extension of [VerneMQ](https://github.com/vernemq/vernemq) with some features for dojot case.
+The Dojot VerneMQ service is an extension of [VerneMQ](https://github.com/vernemq/vernemq) with some
+modifications for Dojot's integration.
 
+# **Configuration**
 
-## **Environment variables**
+There are two configurations for VerneMQ: the one provided by the VerneMQ itself and the Dojot's
+one.
 
-Key                      | Purpose                                                             | Default Value  | Accepted values
------------------------  | --------------------------------------------------------------      | -------------- |------------
-EJBCA_ADDRESS            | Address of the EJBCA broker                                         | "ejbca-wrapper:5583"| IP or DNS
-USE_VMQ_OPERATOR         | Enables the broker for use with the vmq-operator                    | "n"            | y or n
-EXTERNAL_SERVER_HOSTNAME | Server hostname (the host to connect external)                      | "localhost"    | hostname
-EXTERNAL_SERVER_IP       | Server IP (the IP to connect external)                              | ""             | IP
-INTERNAL_DNS             | Internal hostname (used to connect for vk2-bridge and k2v-bridge)   | "vernemq-k8s"  | hostname
-CA_NAME                  | CA Name from EJBCA                                                  | "IOTmidCA"     | string
-CHECK_EXPIRATION_TIME    | Check if the certificate has expired every time defined by this cron expression and renews them if necessary | "0 1 * * *"    | cron schedule expressions
-CHECK_BROKER_CERT_REVOKED_TIME  | Checks if the public certificate of broker has revoked every time defined by this cron expression and renews them if necessary  | "0 */3 * * *" | cron schedule expressions
-CRL_UPDATE_TIME          | Retrieve the new CRL every time defined by this cron expression | "0 */2 * * *" | cron schedule expressions
-CHECKEND_EXPIRATION_SEC  | Check the certificates states and renews them if they expire within the next arg seconds | 43200  | seconds
+## **Dojot**
 
+Key                             | Purpose                                                                                     | Default Value           | Accepted values
+------------------------------- | ------------------------------------------------------------------------------------------- | ----------------------- |-----------------
+CA_NAME                         | CA Name from EJBCA                                                                          | IOTmidCA                | string
+CHECK_EXPIRATION_TIME           | Cron interval for certificate validation, renewing if necessary                             | 0 1 * * *               | cron expressions
```

Comment:
Wasn't this removed?

Reply:
I think that the only thing we changed was the `update` parameter for the CRL retrieval. If we were to remove this here, we should have the sidecar service first, don't we?

Reply:
OK!